doc/release-notes-30352.md

```diff
@@ -0,0 +1,10 @@
+P2P and network changes
+-----------------------
+
+- Pay To Anchor(P2A) is a new standard witness output type for spending,
+  a newly recognised output template. This allows for key-less anchor
```

Comment:
nitty nit:  Although I think it's perfectly clear, I'm not 100% sure `output template` fully aligns with the terminology BIP (https://github.com/murchandamus/bips/blob/2022-04-tx-terminology/bip-tx-terminology.mediawiki).  Maybe `script template`?

Reply:
not sure it's worth the churn, sorry :) leaving as-is